mlflow/store/model_registry/sqlalchemy_store.py

```diff
@@ -1014,3 +1016,71 @@ def delete_model_version_tag(self, name, version, key):
             existing_tag = self._get_model_version_tag(session, name, version, key)
             if existing_tag is not None:
                 session.delete(existing_tag)
+
+    @classmethod
+    def _get_registered_model_alias(cls, session, name, alias):
+        aliases = (
+            session.query(SqlRegisteredModelAlias)
+            .filter(
+                SqlRegisteredModelAlias.name == name,
+                SqlRegisteredModelAlias.alias == alias,
+            )
+            .all()
+        )
```

Comment:
```suggestion
        return (
            session.query(SqlRegisteredModelAlias)
            .filter(
                SqlRegisteredModelAlias.name == name,
                SqlRegisteredModelAlias.alias == alias,
            )
            .first()
        )
```

Can we return `.first()` because `name` and `alias` are a primary key and always unique?